docs/getting-started/start-app.md

```diff
@@ -15,7 +15,7 @@ bin/startup
 ```
 
 (This just runs `foreman start -f Procfile.dev`, for notes on how to install
-Foreman, please see [Other Tools](../installation/others.md))
+Foreman, please see [Other Tools](../installation/others))
```

Comment:
this relative link doesn't work for me :(

ps. a contributor had already submit a PR to fix this https://github.com/thepracticaldev/dev.to/pull/5935 - I think we should merge that

Reply:
Good call, merged it!

Reply:
Oh, sorry for that one, VSCode autocompletes on this, which is technically smart but not what we wanted in this instance. Thanks for merging it @joshpuetz.